src/components/Publish/Preview/index.tsx

```diff
@@ -14,26 +14,28 @@ export default function Preview(): ReactElement {
     async function makeDdo() {
       const asset = (await transformPublishFormToDdo(values)) as AssetExtended
       // dummy BestPrice to trigger certain AssetActions
-      asset.accessDetails = {
-        type: values.pricing.type,
-        addressOrId: ZERO_ADDRESS,
-        templateId: 1,
-        price: `${values.pricing.price}`,
-        baseToken: {
-          address: ZERO_ADDRESS,
-          name: values.pricing?.baseToken?.symbol || 'OCEAN',
-          symbol: values.pricing?.baseToken?.symbol || 'OCEAN'
-        },
-        datatoken: {
-          address: ZERO_ADDRESS,
-          name: '',
-          symbol: ''
-        },
-        isPurchasable: true,
-        isOwned: false,
-        validOrderTx: '',
-        publisherMarketOrderFee: '0'
-      }
+      asset.accessDetails = [
+        {
+          type: values.pricing.type,
+          addressOrId: ZERO_ADDRESS,
+          templateId: 1,
+          price: `${values.pricing.price}`,
+          baseToken: {
+            address: ZERO_ADDRESS,
+            name: values.pricing?.baseToken?.symbol || 'OCEAN',
```

Comment:
name: values.pricing?.baseToken?.name instead of symbol

Reply:
fixed